docs/source/configuration/telemetry/exporters/logging/stdout.mdx

```diff
@@ -326,40 +326,27 @@ Example output with a list of spans:
 
 #### `display_resource`
 
-The `telemetry.logging.stdout.format.json.display_span_list` option is like [`display_current_span`](#displaycurrentspan) but instead of outputting information for the current span, `display_span_list`  outputs information for all spans that an event was raised in.
+The `telemetry.logging.stdout.format.json.display_resource` option configures whether resources configured in `router.yaml` are attached to and displayed in log messages. By default, `display_resource` is `true`.
```

Comment:
What does "attached to" mean in this context? If it's just restating "displayed in", let's remove it. If attaching is a separate thing, can it be included in the example below?